benchmark/generate_workload.py

```diff
@@ -38,7 +38,7 @@ def generateWorkload(query_name, field="id.orig_h", runs=1000, seed=42):
     random.seed(seed)
     for i in range(runs):
         uniqueVal = random.choice(uniqueVals)
-        workload.append({'query': query_name + " " + field, 'arguments': [uniqueVal]})
+        workload.append({'query': query_name , 'arguments': [uniqueVal]})
```

Comment:
If you're going to change this line to omit the field name, then perhaps change line 49 below so that the input query name is "search id.orig_h", to avoid breaking compatibility with other scripts that assume the query name is "search id.orig_h" in the output.